melusine/config/config.py

```diff
@@ -208,5 +208,72 @@ def switch_config(new_config):
     config._switch_config(new_config)
 
 
+def ensure_retro_compatibility(config_dict):
+    """ """
+
+    def check_key(d, path):
```

Comment:
Hello, can you add params type and a quick description of def ensure_retro_compatibility?  :)